src/widgets/Footer/components/OrderProcessing.tsx

```diff
@@ -0,0 +1,14 @@
+const OrderProcessing = () => {
+  return (
+    <div className="flex flex-col justify-start mb-13">
+      <div className="font-roboto text-xs font-medium text-text-tertiary">
+        ORDER PROCESSING
+      </div>
+      <div className="flex flex-row items-center gap-1 mt-3 font-normal text-base text-text-secondary">
```

Comment:
"text-text-secondary" I see that you duplicate text-text. Maybe you can make it shorter to just "text-secondary"?

Reply:
Text-text-secondary`s already become neutral-50 after the designer's changes